### ISSUE TITLE
WebUI displays incorrect trump suit on reset

### ISSUE THREAD

**Issue by m-rots** [intelligent-systems-course/schnapsen-old#6]
In #1 the webUI now also displays the trump suite in phase 2.
However, when resetting the game with SHIFT+R, the trump suit of the previous game is displayed.

![image](https://user-images.githubusercontent.com/29630936/85201169-79e8ee80-b2fd-11ea-8c22-3c90054be70c.png)

The previous game had hearts as trump suit, as displayed on the right. While the new game has clubs as trump suit, as displayed on the left.

**Comment by miselico** [comment]
@hasan-sh any easy way to cover this corner case?

**Comment by hasan-sh** [comment]
The PR that was merged into this repo's master was the first one i made, however, the later one, that solved some other issues, is still a PR at https://github.com/PaulUrsulean/schnapsen/pull/2 and i cannot reproduce the problem with the later. So can we merge it into this repo? I also see you wanted to cherry-pick them with Paul.

**Comment by miselico** [comment]
As we have only a week left in the current course edition, I suggest we look into it later. This is a rather rare situation, I think. I didn't even know the shift-R shortcut existed. OK?

**Comment by hasan-sh** [comment]
> As we have only a week left in the current course edition, I suggest we look into it later. This is a rather rare situation, I think. I didn't even know the shift-R shortcut existed. OK?

I totally agree. For now, if someone encountered this, just terminate from the terminal and rerun it.
